packet_test.go

```diff
@@ -474,34 +518,44 @@ func BenchmarkMarshalInit(b *testing.B) {
 	})
 }
 
+func benchMarshalFX(b *testing.B, packet sshfx.PacketMarshaller) {
+	conn := &conn{
+		Writer: ioutil.Discard,
+	}
+
+	// Intentionally set to the default value in client.go.
+	buf := make([]byte, 1<<15)
```

Comment:
So, trying to fix things so `WorstCase` is forced to do an allocation each loop, I realized something: because of how `MarshalPacket()` works, it doesn’t matter what size the `Data` field is… it’s just passed through to the `payload []byte` return value. :laughing: